### PR TITLE
Updating Trento Checks

### PR DESCRIPTION
**Pull request by abravosuse**
This PR is to be reviewed but cannot be merged/published until the new version of Trento is out.

It adds a new section describing how to update the configuration checks in a Trento installation.

### CODE REVIEW

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -2393,6 +2393,34 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
       </step>
     </procedure>
 
+  </section>
+
+    <section xml:id="sec-trento-updating-trento-checks">
+    <title>Updating Trento Checks</title>
+    <para>Configuration checks are an integral part of &t.server; and are delivered as a separate component of it. This allows customers
+    to update the checks catalog in their setup whenever updates to existing chekcs and/or new checks are released, without
+    having to wait to a full mew version release cycle.</para>
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:2402]:
```suggestion
    <para>Configuration checks are an integral part of the &t.server;. The checks are delivered as a separate component of the &t.server;. This allows customers
    to update the checks catalog in their setup whenever updates to existing checks and new checks are released, without
    waiting for a new version release cycle.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -2393,6 +2393,34 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+         <para> The procedure to update the configuration checks depends on the &t.server; deployment type: &k8s;, systemd or containerized.</para>
+     <para>In a K8s; deployment, checks are delivered as a container image and you can
+      use Helm with the following options to pull the latest one available into your setup:</para>
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:2405]:
```suggestion
         <para> The procedure of updating the configuration checks depends on the &t.server; deployment type: &k8s;, systemd or containerized.</para>
     <para>In a k8s; deployment, checks are delivered as a container image and you can
      use Helm with the following options to pull the latest one available into your setup:</para>
```

File: trento/xml/article_sap_trento.xml [hunk 3]
```diff
@@ -2393,6 +2393,34 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+<programlisting>
+    helm ... \
+   --set trento-wanda.checks.image.tag=latest \
+   --set trento-wanda.checks.image.repository=registry.suse.com/trento/trento-checks  \
+   --set trento-wanda.checks.image.pullPolicy=Always \
+   ...
+</programlisting>
+     <para>In a systemd deployment, checks are delivered as an rpm package and you can 
+      use zypper to bring your checks catalog to the latest version avaialble:</para>
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:2414]:
```suggestion
     <para>In a systemd deployment, checks are delivered as an RPM package, and you can 
      use zypper to update your checks catalog to the latest version:</para>
```

File: trento/xml/article_sap_trento.xml [hunk 4]
```diff
@@ -2393,6 +2393,34 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+        <screen>&prompt.user;sudo zypper ref
+&prompt.user;sudo zypper update trento-checks</screen>
+    <para>In a containerized deployment, checks are also delivered as a container image and
+     you can user docker to pull the latest version available into the trento-checks volume that was
+     created during the installation process:</para>
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:2419]:
```suggestion
    <para>In a containerized deployment, checks are delivered as a container image, and
     you can user Docker to pull the latest version into the trento-checks volume that was
     created during the installation process:</para>
```

**Review by abravosuse — COMMENTED** [review]
All good. Thanks for the suggestions @dmpop !

**Review by tomschr — APPROVED** [review]

**Pull request merged**